docs/intro.md

```diff
@@ -0,0 +1,127 @@
+# A brieft introduction to HADDOCK3
+
+HADDOCK3 is the next generation of integrative modelling software in the
+long-lasting HADDOCK project. It represents a complete rethinking and rewriting
+of the HADDOCK2.X series, implementing a new way to interact with HADDOCK and
+offering new features to users who can now define custom workflows. 
+
+In the previous HADDOCK2.x versions, users had access to a highly
+parameterisable yet rigid simulation pipeline composed of three mandatory steps:
+`it0`, `it1`, and `itwater`.
+
+![HADDOCK 2.x workflow](figs/HADDOCK2-stages.png)
+
+In HADDOCK3, users have the freedom to configure docking workflows into
+functional pipelines by combining the different HADDOCK3's modules, thus
+adapting the workflows to their projects. HADDOCK3 has therefore developed to
+truthfully work like a puzzle of many pieces (simulation modules) that users can
+combine freely. To this end, the “old” HADDOCK machinery has been modularised,
+and several new modules added, including third-party software additions. As a
+result, the modularisation achieved in HADDOCK3 allows users to duplicate steps
+within one workflow (e.g., to repeat twice the `it1` stage of the HADDOCK2.x
+rigid workflow).
+
+Note that, for simplification purposes, at this time, not all functionalities of
+HADDOCK2.x have been ported to HADDOCK3, which does not (yet) support NMR RDC,
+PCS and diffusion anisotropy restraints, cryo-EM restraints and coarse-graining.
+Any type of information that can be converted into ambiguous interaction
+restraints can, however, be used in HADDOCK3, which also supports the
+*ab initio* docking modes of HADDOCK.
+
+![HADDOCK3 workflow](figs/HADDOCK3-workflow-scheme.png)
+
+To keep HADDOCK3's modules organised, we catalogued them into several
+categories. But, there are no constraints on piping modules of different
+categories.
+
+The main module's categories are "topology", "sampling", "refinement",
+"scoring", and "analysis". There is no limit to how many modules can belong to a
+category. Modules are added as developed, and new categories will be created
+if/when needed. You can access the HADDOCK3 documentation page for the list of
+all categories and modules. Below is a summary of the available modules:
```

Comment:
as this is not crucial for the user, I would maybe reduce the lenght of this paragraph

Reply:
They are for the understanding of the project and the different modules. That's how we are presenting them in talks. Let's keep it for now and let the text evolve as we read it more times in the future. Many thanks!